target/linux/realtek/files-5.10/drivers/i2c/busses/i2c-rtl9300.c

```diff
@@ -0,0 +1,484 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+#include <linux/module.h>
+#include <linux/of_platform.h>
+#include "i2c-rtl9300.h"
+
+#define REG(x)		(i2c->base + x + (i2c->scl_num ? 0x1c : 0))
```

Comment:
What is i2c? There's no context as to what it is, here with this macro.

Reply:
I'll change this to #define REG(i, x)		(i->base + x + (i->scl_num ? 0x1c : 0))
and elsewhere REG(x) -> REG(i,x)
That was quite minimalistic, I agree. I will also pick the 0x1c from a variable in i2c.